project/SmallJDK.scala

```diff
@@ -26,7 +27,9 @@ object SmallJDK {
   private val JDK_MODULES = Seq(
     "java.naming",
     "java.net.http",
+    "java.prefs",
```

Comment:
why?

Reply:
Thanks for reporting https://github.com/oracle/graal/pull/12245 to GraalVM guys, Pavel!